src/cata_tiles.h

```diff
@@ -6,8 +6,13 @@
 #include <wordexp.h>
 #endif
 
-#include "SDL2/SDL.h"
-#include "SDL2/SDL_ttf.h"
+#if (defined OSX_SDL_FW)
+#   include "SDL.h"
+#   include "SDL_ttf.h"
+#else
+#   include "SDL2/SDL.h"
```

Comment:
Instead of having these ifdefs scattered around, can you append "SDL2/" to the end of the paths you're setting in the Makefile?